src/SignalR/clients/ts/FunctionalTests/SignalR.Npm.FunctionalTests.npmproj

```diff
@@ -3,17 +3,16 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
-    <IsTestProject>false</IsTestProject>
+    <IsTestProject>true</IsTestProject>
```

Comment:
Curious: Any idea why this was `false` before?

Reply:
Because arcade broke the tests and we had to disable them to make the builds pass